### ISSUE TITLE
import specific ufsda stage functions in ufsda atm prep script

### ISSUE THREAD

**Issue by RussTreadon-NOAA** [NOAA-EMC/global-workflow#1262]
**Description**
Specify the functions to import from ufsda.stage in UFS-DA atmospheric prep script.

**Requirements**
GDASApp PR [#278](https://github.com/NOAA-EMC/GDASApp/pull/278) removes stage from `ufsda/__init__.py`.   As a result, global-workflow `scripts/exgdas_global_atmos_analysis_prep.py` must specify the specific stage functions it uses.   GDASApp includes the option to run ctests with g-w.   Test `test_gdasapp_atm_jjob_var_prep` fails without updating `exgdas_global_atmos_analysis_prep.py`.

**Acceptance Criteria (Definition of Done)**
GDASApp g-w ctest `test_gdasapp_atm_jjob_var_prep` should pass.

**Dependencies**
Closure of this issue requires GDASApp PR [#278](https://github.com/NOAA-EMC/GDASApp/pull/278) to be merged into GDASApp `develop`.   Once this is done, the GDASApp has in g-w `sorc/checkout.sh` and `Externals.cfg` should be updated along with the change to `scripts/exgdas_global_atmos_analysis_prep.py`.

**Comment by RussTreadon-NOAA** [comment]
Work for this development will be done in RussTreadon-NOAA/global-workflow branch [`feature/ufsda_stage`](https://github.com/RussTreadon-NOAA/global-workflow/tree/feature/ufsda_stage).

**Comment by RussTreadon-NOAA** [comment]
@WalterKolczynski-NOAA  and @KateFriedman-NOAA , not sure why automated test [Install pygw and run tests with pytests (3.7)](https://github.com/RussTreadon-NOAA/global-workflow/actions/runs/3971382519/jobs/6808209182#logs) failed with a9c3d0e.

The log file shows the following test failing with two tests skipped.
```
src/tests/test_configuration.py::test_configuration_config_files FAILED
src/tests/test_configuration.py::test_parse_config1 SKIPPED (fails i...) [ 37%]
src/tests/test_configuration.py::test_parse_config2 SKIPPED (fails i...) [ 41%]
```
The log file contains 
```
src/tests/test_configuration.py:149: AssertionError
=========================== short test summary info ============================
FAILED src/tests/test_configuration.py::test_configuration_config_files - AssertionError: assert ['/tmp/pytest...config.file1'] == ['/tmp/pytest...config.file0']
```

**Comment by RussTreadon-NOAA** [comment]
@CoryMartin-NOAA 's PR #1106 has a similar failure in the automated pytests ([see here](https://github.com/NOAA-EMC/global-workflow/actions/runs/3971318267/jobs/6808072977)).

**Comment by WalterKolczynski-NOAA** [comment]
The pytests are just failing in general for everyone. Started a couple weeks ago. We're not sure why and may have to turn them off until we figure it out.

**Comment by RussTreadon-NOAA** [comment]
Thank you, @WalterKolczynski-NOAA .  I'll hit pause on digging into what's going on.
